tests/share/search/index_strategy/_with_real_services.py

```diff
@@ -95,31 +86,35 @@ def _assert_happypath_with_daemon(self, messages_chunk, expected_doc_count):
 
     def _assert_setup_happypath(self):
         # initial
-        assert not self.current_index.pls_check_exists()
-        index_status = self.current_index.pls_get_status()
-        assert not index_status.creation_date
-        assert not index_status.is_kept_live
-        assert not index_status.is_default_for_searching
-        assert not index_status.doc_count
-        # create index
-        self.current_index.pls_create()
-        assert self.current_index.pls_check_exists()
-        index_status = self.current_index.pls_get_status()
-        assert index_status.creation_date
-        assert not index_status.is_kept_live
-        assert not index_status.is_default_for_searching
-        assert not index_status.doc_count
+        for _index in self.index_strategy.each_subnamed_index():
+            assert not _index.pls_check_exists()
+            index_status = _index.pls_get_status()
+            assert not index_status.creation_date
+            assert not index_status.is_kept_live
+            assert not index_status.is_default_for_searching
+            assert not index_status.doc_count
+        for _index in self.index_strategy.each_subnamed_index():
```

Comment:
nit: maybe add a comment before each for loop to explain what you're about to test? These four for loops all kinda look the same at first glance.